lib/foreman_scap_client/client.rb

```diff
@@ -36,7 +36,17 @@ def scan_command
                                else
                                  ''
                                end
-      "oscap xccdf eval #{fetch_remote_resources} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+      "oscap xccdf eval #{fetch_remote_resources} #{local_files_subcommand} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+    end
+
+    def local_files_subcommand
+      supports_local_file_option ? '--local-files /root' : ''
+    end
+
+    def supports_local_file_option
+      # OpenSCAP 1.3.6 and newer requires the `--local-files` option to use local copies of remote SDS components
+      version = `rpm -q openscap`.split('-')[1]
+      Gem::Version.new(version) >= Gem::Version.new('1.3.6') && !config[:fetch_remote_resources]
```

Comment:
I'd keep the `config` part out since that doesn't have anything to do with support or not. That will remain even if support for version < 1.3.6 is dropped.